dir.props

```diff
@@ -14,6 +14,11 @@
     <RoslynPackageName>Microsoft.Net.ToolsetCompilers</RoslynPackageName>
   </PropertyGroup>
 
+  <!-- MSBuild will set NuGetRuntimeIdentifier to be 'win' so we pre set it correctly here -->
+  <PropertyGroup>
+    <NuGetRuntimeIdentifier Condition="'$(NuGetRuntimeIdentifier)'==''">None</NuGetRuntimeIdentifier>
```

Comment:
If we don't specifically set this, then it would be defaulted to win by MSBuild targets. The property is already being set [in the targets that use it](https://github.com/dotnet/buildtools/blob/master/src/Microsoft.DotNet.Build.Tasks/PackageFiles/packageresolve.targets#L61) the problem is that it gets set by MSBuild earlier in the build process. That is why what I did is to set the default in here to be None, and then only override it on the projects that actually need it.